sdks/python/apache_beam/utils/retry.py

```diff
@@ -57,6 +60,18 @@
 # pylint: enable=wrong-import-order, wrong-import-position
 
 _LOGGER = logging.getLogger(__name__)
+_RETRYABLE_REASONS = [
+    "rateLimitExceeded", "quotaExceeded", "internalError", "backendError"
```

Comment:
why do we need to retry on `quotaExceeded`? Once the quota exceeded it needs a manual intervention or back-off of substantial amount of time.

Reply:
For some cases, the quota exceeded error could be transient.

Reply:
Removed